jbpm-services/jbpm-kie-services/src/test/java/org/jbpm/kie/services/test/UserTaskServiceImplTest.java

```diff
@@ -893,7 +893,7 @@ public void testTaskOperationWithUserOrWithIdentityProvider() {
         assertEquals(TaskEvent.TaskEventType.STOPPED, auditTasks.get(3).getType());
         assertEquals("salaboy", auditTasks.get(3).getUserId());
         assertEquals(TaskEvent.TaskEventType.UPDATED, auditTasks.get(4).getType());
-        assertEquals("salaboy", auditTasks.get(4).getUserId());
+        assertEquals("Administrator", auditTasks.get(4).getUserId());
```

Comment:
Is this expected behavior? I mean, after making a call to `saveContent`, task owner changes from current user `salaboy` to `Administrator` (or whoever it's set in the `org.jbpm.ht.admin.user` property). Is this ok or am I missing anything here?